src/components/Graph/MainGraph.js

```diff
@@ -89,37 +90,50 @@ class MainGraph extends Component {
                 const copy = Array.from(
                     dataset[scenarioList[i].key][severityList[i].key][stat.key].sims);
 
-                // filter down sims on reproductive number
-                const newSeries = copy.filter(s => { return (s.r0 > r0[0] && s.r0 < r0[1]) });
-                    
-                const filteredSeriesForStatThreshold = newSeries.map( s => {
+                // filter down sims on r0
+                const r0min = r0selected[0], r0max = r0selected[1];
+                const series = copy.filter(s => { 
+                    return (s.r0 > r0min && s.r0 < r0max)})
+                    .slice(0, numDisplaySims);
+             
+                // setting default smart threshold based on seriesMin 
+                const filteredSeriesForStatThreshold = series.map( s => {
                     const newS = {...s}
                     newS.vals = s.vals.slice(idxMin, idxMax)
                     return newS
                 });
+                // array of all peaks in series
                 const seriesPeaks = filteredSeriesForStatThreshold.map(sim => max(sim.vals));
                 const [seriesMin, seriesMax] = getRange(seriesPeaks);
+                // ensures side by side y-scale reflect both series
                 if (seriesMin < sliderMin) sliderMin = seriesMin
                 if (seriesMax > sliderMax) sliderMax = seriesMax
                 // default smart value for statThreshold calculation
                 if (i === 0 && seriesMin < seriesMax/2) statThreshold = seriesMin;
                 if (i === 0 && seriesMin >= seriesMax/2) statThreshold = seriesMax/2;
                 const simsOver = returnSimsOverThreshold(
-                    newSeries, statThreshold, this.state.allTimeDates, dateThreshold);
-                if (i === 0) brushSeries = newSeries
+                    series, statThreshold, this.state.allTimeDates, dateThreshold);
+                // brush visual only based on first scenario, for simplicity
+                if (i === 0) brushSeries = series
 
-                const percExceedence = simsOver / newSeries.length;
-                percExceedenceList.push(percExceedence)
-
-                const filteredSeries = newSeries.map( s => { const newS = {...s}
+                // filtering based on date
+                // only dateRange change needs this ---> 
+                const filteredSeries = series.map( s => { const newS = {...s}
                     newS.vals = s.vals.slice(idxMin, idxMax)
                     return newS
                 })
-                filteredSeriesList.push(filteredSeries)
                 
+                filteredSeriesList.push(filteredSeries)
+                // console.log('on brush change', filteredSeries.map(sim => {return `${sim.name}: ${sim.r0}`}))
+
+                // TODO: may be problematic
+                const percExceedence = simsOver / filteredSeries.length;
```

Comment:
dunno what JS does when dividing by 0 but you may want to add a defensive check here that you're not trying to divide by 0 if `filteredSeries.length` is null